### DISCUSSION TITLE
Choice of Antenna

### DISCUSSION THREAD

**Discussion by FinSmy** [PE5PVB/SI4684-DAB-Receiver#8] [General]
Hi there, 

One thing which I am interested in here is the choice of antenna for receiving of DAB/FM frequencies with adequate bandwidth. I don't think you mention in particular which antenna you use and it would be interesting to know. More generally, it would also be interesting to know is how to go about selecting antennas for these kinds of projects since cost/size/connector are all considerations. 

Thanks in advance

**Comment by PE5PVB** [comment]
Any Band 3 antenna with 50/75 ohm impedance should work.
